addons/event_sms/models/__init__.py

```diff
@@ -2,5 +2,4 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
Commit message:

> '_sms_get_number_fields' was introduced at odoo/odoo@ XXXX to have a generic
implementation of finding partners on a record. Since then another version
has been added directly in 'phone' module, using '_phone_get_partner_fields'.
This is now generic for all models when phone is installed, meaning the sms
specific one is not necessary anymore. This allows to simplify model custom
code when dealing with phone and sms.

- in the 'phone' module -> it's phone_validation, right?
- using '_phone_get_partner_fields' -> I think you mean '_phone_get_number_fields' here